pkgs/top-level/all-packages.nix

```diff
@@ -15577,10 +15577,16 @@ with pkgs;
   };
 
   ocaml-ng = callPackage ./ocaml-packages.nix { };
-  ocaml = ocamlPackages.ocaml;
 
+  # Ocaml package sets
+  ocamlPackages_4_13 = recurseIntoAttrs ocaml-ng.ocamlPackages_4_13;
```

Comment:
I don't think we need to add 4.13 to top-level: `ocamlPackages` is already at 4.14.

```suggestion
```

Reply:
Agreed. As it seems to be duplicating builds for OCaml (4 and 5). Makes sense to limit it the most.

Reply:
@vbgl Thoughts?